app/app.js

```diff
@@ -21,6 +21,7 @@ window.d3 = require('d3'); // Required for nvd3
 require('./common/wrapper/nvd3-wrapper');
 require('angular-nvd3/src/angular-nvd3');
 require('angular-cache');
+require('angular-uuid');
```

Comment:
Sorry, should have mentioned that we should probably generate this at the backend since we no longer need the API user to provide/ change this?